src/networks.json

```diff
@@ -498,5 +498,15 @@
     "multicall": "0x537004440ffFE1D4AE9F009031Fc2b0385FCA9F1",
     "rpc": ["https://rpc.moonriver.moonbeam.network"],
     "explorer": "https://blockscout.moonriver.moonbeam.network/"
+  },
+    "336": {
+    "key": "336",
+    "name": "Shiden (Kusama)",
+    "shortName": "Shiden",
+    "chainId": 336,
+    "network": "mainnet",
+    "multicall": "0x8AfCFB1757489B89c73fd2E0bd8fF0a5D985a8d9",
```

Comment:
I am getting this error when i try to get balance of 0x8AfCFB1757489B89c73fd2E0bd8fF0a5D985a8d9 at block `1` maybe this is not a full archive node??
```
 (body="{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32603,\"message\":\"fetch runtime chain id failed: Application(Execution(Other(\\\"Exported method EthereumRuntimeRPCApi_account_basic is not found\\\")))\"},\"id\":44}\n", error={"code":-32603}, requestBody="{\"method\":\"eth_getBalance\",\"params\":[\"0x8afcfb1757489b89c73fd2e0bd8ff0a5d985a8d9\",\"0x1\"],\"id\":44,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://rpc.shiden.astar.network:8545", code=SERVER_ERROR, version=web/5.4.0)
 ```